### DISCUSSION TITLE
0.12-rc

### DISCUSSION THREAD

**Discussion by xz-dev** [DUpdateSystem/UpgradeAll#45] [General]
1. 修复春节彩蛋
2. 同步台湾翻译
3. 修复一处空值导致的崩溃
4. 更新依赖

<hr /><em>This discussion was created from the release <a href='https://github.com/DUpdateSystem/UpgradeAll/releases/tag/0.12-rc'>0.12-rc</a>.</em>

**Comment by JohnsonRan** [comment]
好耶！
